2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2020 @@
+---
+abstract: |
+    Research software has become a central asset in academic research. It
+    optimizes existing and enables new research methods, implements and
+    embeds research knowledge, and constitutes a valid research product in
+    itself. Research software must be sustainable in order to understand,
+    replicate, reproduce, and build upon existing research or conduct new
+    research effectively. In other words, software must be available,
+    discoverable, usable, and adaptable to new needs, both now and in the
+    future. Research software therefore requires an environment that
+    supports sustainability.
+
+    Hence, a change is needed in the way research software development and
+    maintenance are currently motivated, incentivized, funded, structurally
+    and infrastructurally supported, and legally treated. Failing to do so
+    will threaten the quality and validity of research. In this paper, we
+    identify challenges for research software sustainability for Germany and
+    beyond, in terms of motivation, selection, research software engineering
+    personnel, funding, infrastructure and legal aspects. We address
+    political and academic decision makers to increase awareness of the
+    importance and needs of sustainable research software practices. In
+    particular, we recommend strategies and measures to create an
+    environment for sustainable research software, with the ultimate goal to
+    ensure that software-driven research is valid, reproducible and
+    sustainable. Software should be recognized as a first class citizen in
+    research. This paper is the outcome of two workshops run in Germany in
+    2019, at deRSE19 - the first International Conference of Research
+    Software Engineers in Germany - and a dedicated DFG-supported follow-up
+    workshop in Berlin.
+author:
+# Main authors (editorial team, alphabetical by last name)
+- 'Hartwig Anzt 1,2,†'
+- 'Felix Bach 1,†'
+- 'Stephan Druskat 3,4,5,†'
+- 'Frank Löffler 6,7,8,†'
+- 'Axel Loewe 9,*,†'
+- 'Bernhard Y. Renard 10,11,†'
+- 'Gunnar Seemann 12,13,*,†'
+- 'Alexander Struck 14,†'
+# Additional authors (alphabetical by last name)
+- 'Elke Achhammer 15'
+- 'Piush Aggarwal 16'
+- 'Franziska Appel 17'
+- 'Michael Bader 18'
+- 'Lutz Brusch 19'
+- 'Peter Ebert 20'
+- 'Bernd Flemisch 21'
+- 'Sven Friedl 22'
+- 'Maximilian D. Funk 23'
+- 'Florian Goth 24'
+- 'Jean-Noël Grad 25'
+- 'Sibylle Hermann 26'
+- 'Florian Hohmann 27'
+- 'Dominik Kutra 28'
+- 'Jan Linxweiler 29'
+- 'Thilo Muth 10,30'
+- 'Wolfgang Peters-Kottig 31'
+- 'Fabian Rack 32'
+- 'Fabian H.C. Raters 33'
+- 'Stephan Rave 34'
+- 'Guido Reina 35'
+- 'Timo Ropinski 36,37'
+- 'Joerg Schaarschmidt 38'
```

Comment:
```suggestion
- 'Joerg Schaarschmidt 38'
- 'Heidi Seibold 42, 43, 44'
```

Reply:
:point_up_2: of course only if you deem my contributions valuable enough for coauthorship ;)